Dockerfile

```diff
@@ -21,6 +21,14 @@ RUN wget https://mirrors.ocf.berkeley.edu/gnu/autoconf/autoconf-2.71.tar.xz \
     && cp /usr/local/bin/autoconf /usr/bin/autoconf \
     && rm -rf autoconf-2.71
 
+# install libtool 2.5.4, required for emscripten support
```

Comment:
Could you please update the comment mentioning that it is required for `ngspice` package specifically, so that we can remove it when we are not building ngspice in the repository? (#4918).

Reply:
Done